packages/backend-common/src/logging/rootLogger.test.ts

```diff
@@ -29,4 +29,79 @@ describe('rootLogger', () => {
       expect.stringContaining('testing'),
     );
   });
+
+  describe('createRootLoger', () => {
+    beforeEach(() => {
```

Comment:
maybe `afterEach` instead? :P

Reply:
Wanted to clean this up first before running tests in case it's set in someones env already.